File: c/fizzbuzz/fizzbuzz.c
#include<stdio.h>

int main(void){
    int ix;
    for(ix=1;ix<101;ix++){
        if(ix%3==0){
            printf("fizz");
        }
        if(ix%5==0){
            printf("buzz");
        }
        if(ix%3!=0 && ix%5!=0){
            printf("%d",ix);
        }
        printf("\n");
    }
    return 0;
}

File: Python/fizzbuzz.py
#coding:utf-8

i = 1 

while True:
    str = ''
    if i > 100:
     break
    if i % 3 == 0:
        str = 'fizz'
    if i % 5 == 0:
        str = str + 'buzz'
    if str == '':
        str = "%d" % i
    print str
    i += 1
